src/diff-snapshot.js

```diff
@@ -20,6 +20,61 @@ const pixelmatch = require('pixelmatch');
 const mkdirp = require('mkdirp');
 const { PNG } = require('pngjs');
 
+/**
+ * Helper function to create reusable image resizer
+ */
+const createImageResizer = (width, height) => (source) => {
+  const resized = new PNG({ width, height, fill: true });
+  PNG.bitblt(source, resized, 0, 0, source.width, source.height, 0, 0);
+  return resized;
+};
+
+/**
+ * Fills diff area with black transparent color for meaningful diff
+ */
+/* eslint-disable no-plusplus, no-param-reassign, no-bitwise */
```

Comment:
I am okay with disabling `no-plusplus` since in a loop that is a pretty standard way to iterate that has been around since the beginning of programming. And `no-bitwise` since you do need it.

However you should definitely not disable `no-param-reassign`.

Reply:
`no-param-reassign` required to mute errors for reassigning `image.data` value